code/core/src/builder-manager/index.ts

```diff
@@ -28,6 +28,15 @@ const isRootPath = /^\/($|\?)/;
 let compilation: Compilation;
 let asyncIterator: ReturnType<StarterFunction> | ReturnType<BuilderFunction>;
 
+export const BROWSER_TARGETS = [
+  'chrome131',
+  'edge134',
+  'firefox136',
+  'safari18.3',
+  'ios18.3',
+  'opera117',
+];
```

Comment:
This should be in the shared dir, maybe?

Reply:
Yeah I dont have a strong opinion on that.

Reply:
We need to (and i did it).

If the script to build needs this file, and that refers to dist files... that can't work.
So I moved these constants into a file without dependencies.

Reply:
Thank you ❤️